### PR TITLE
fix for the last digit and the last digit list

### PR DESCRIPTION
**Pull request by rupato-deriv**

### CODE REVIEW

**Review by maryiafrantsava-binary — APPROVED** [review]

File: src/botPage/bot/TradeEngine/Ticks.js [hunk 1]
```diff
@@ -58,11 +58,13 @@ export default Engine =>
             );
         }
         getLastDigit() {
-            return new Promise(resolve => this.getLastTick().then(tick => resolve(getLastDigit(tick))));
+            return new Promise(resolve => {
+                this.getLastTick().then(tick => resolve(getLastDigit(tick.toFixed(this.getPipSize()))));
```
Comment (sara-fs) [inline src/botPage/bot/TradeEngine/Ticks.js:62]: I think there is a problme with toFixed method and we have to change it. The problme is that when the argument is 2 it functions well but when the argument is 1 for example it rounds the tick digits and then get the last digit. 
Another problem is that toFixed returns a string and if you want to sum up the last digit block with a number, the result would be wrong. Could you plz test these things? I have the same issue and working on them on the other card Thanks

File: src/botPage/bot/TradeEngine/Ticks.js [hunk 2]
```diff
@@ -51,21 +52,26 @@ export default Engine =>
                 this.$scope.ticksService.request({ symbol: this.symbol }).then(ticks => {
                     let lastTick = raw ? getLast(ticks) : getLast(ticks).quote;
                     if (toString && !raw) {
```
Comment (sara-fs) [inline src/botPage/bot/TradeEngine/Ticks.js:54]: So you have to remove toString from here. toString is being repeated 2 times

File: src/botPage/bot/TradeEngine/Ticks.js [hunk 3]
```diff
@@ -51,21 +52,26 @@ export default Engine =>
...
+                        if (!toString) {
+                            lastTick = parseFloat(lastTick);
```
Comment (sara-fs) [inline src/botPage/bot/TradeEngine/Ticks.js:56]: I think we can revert this part of code to the previous one. No need to change it ..

**Review by sara-fs — DISMISSED** [review]

File: src/botPage/common/TicksService.js [hunk 4]
```diff
@@ -58,19 +58,26 @@ export default class TicksService {
...
         }
         return this.active_symbols_promise;
     }
+    getActiveSymbols = () =>
+        new Promise(resolve => {
+            this.api.events.on('authorize', async () => {
+                // eslint-disable-next-line camelcase
+                this.api.send({ active_symbols: 'brief' }).then(({ active_symbols }) =>
+                    // eslint-disable-next-line camelcase
+                    resolve(active_symbols)
```
Comment (sara-fs) [inline src/botPage/common/TicksService.js:77]: Just to make sure.. `active_symbols` or `active_symbols.active_symbols` should be resolved here?

Comment (rupato-deriv) [reply src/botPage/common/TicksService.js:77]: yes active_symbols.active_symbols to be resolved here

**Review by sara-fs — APPROVED** [review]

**Pull request merged**